test-utils/testlib/src/standard_test_cases.rs

```diff
@@ -847,19 +847,19 @@ fn assert_access_key(
     user: &dyn User,
 ) {
     let key = if cfg!(feature = "protocol_feature_access_key_nonce_range") {
-            let mut key = access_key.clone();
-            let block = user.get_block_by_hash(result.transaction_outcome.block_hash);
-            if let Some(b) = block {
-                key.nonce = (b.header.height - 1) * AccessKey::ACCESS_KEY_NONCE_RANGE_MULTIPLIER;
-            }
-            key
-        } else  {
-            let _ = (user, result);
-            access_key.clone()
+        let mut key = access_key.clone();
+        let block = user.get_block_by_hash(result.transaction_outcome.block_hash);
+        if let Some(b) = block {
+            key.nonce = (b.header.height - 1) * AccessKey::ACCESS_KEY_NONCE_RANGE_MULTIPLIER;
         }
+        key
+    } else {
+        let _ = (user, result);
```

Comment:
```suggestion
        let _ = (user, result);
```

You no longer need this